pxtsim/debugger.ts

```diff
@@ -103,6 +103,9 @@ namespace pxsim {
                             preview: RefObject.toDebugString(v)
                         }
                     }
+                    if (v._width && v._height) {
+                        return { text: "Image " + v._width + 'x' + v._height }
```

Comment:
I think we should have a meta comment on the type (e.g. debugstring="Image $_width x $_height") that gets expanded at runtime. Also allows for localization.

Reply:
you are mapping any object that has those 2 fields.

Reply:
For images, I think we should extract the image data and display it in the debugger. Do we have the full buffer of the image?

Reply:
I agree it might be interesting to show the image, same as Sprites properties, clicking on the variables list.

> you are mapping any object that has those 2 fields.

All RefObjects are being handled by the previous case (see line 99). And in any case, if an object has a field property "width", it will be inside 'fields'.

Reply:
just do v._width + "x" + v._height, drop the "image".